lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -418,11 +418,8 @@ sub serial_console_diag_banner {
     # max_length - length of the text - 4x2 dividing spaces
     my $symbol_fill = ($max_length - length($input_text) - 8) / 2;
     $input_text = '#' x $symbol_fill . ' ' x 4 . $input_text . ' ' x 4 . '#' x $symbol_fill;
```

Comment:
there's still an `#` here. Is it fine?

Reply:
> There's still an `#` here. Is it fine? So was the error in wait_serial only about the fill_line?
I think yes according to the issue we found.
It it fine as only  has 1 line, and the VRs can not product the race condition.